tests/subsys/fw_info/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
   fw_info.core:
     platform_allow: nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf52dk_nrf52832 nrf51dk_nrf51422
-      nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuappns
+      nrf5340dk_nrf5340_cpuapp
```

Comment:
Wouldn't we want to check NS compilations as well?

Reply:
NS firmware doesn't have the right privileges for the tests this is performing.